### PR TITLE
[DOCS-8748] Reorder audit trail events

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do? What is the motivation?

No new content added. 

- Reorder Audit Trail events for each product so that there is some logical sense.
- Couple of small edits.

[DOCS-8748](https://datadoghq.atlassian.net/browse/DOCS-8748)

### Merge instructions

- [x] Please merge after reviewing

### Additional notes

[DOCS-8748]: https://datadoghq.atlassian.net/browse/DOCS-8748?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/account_management/audit_trail/events.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ See the [Audit Trail documentation][2] for more information on setting up and co
 | [Custom metrics][113] | A user created, modified, or deleted a custom metric | `@evt.name:APM @action:(created OR modified OR deleted) @asset.type:custom_metrics` |
 | [Facet][21] | A user created, modified, or deleted a [facet][22] and the previous and/or new values for the facet configuration. | `@evt.name:APM @asset.type:facet` |
 | [Primary operation name][23] | A user created, modified, or deleted the [primary operation name][24] of a service and the previous and/or new values for the configuration. | `@evt.name:APM @asset.type:service_operation_name` |
-| [Second Primary tag][25] | A user added, modified, or deleted the [second primary tag][26] and the previous and/or new values for the configuration.  | `@evt.name:APM @asset.type:second_primary_tag` |
+| [Second primary tag][25] | A user added, modified, or deleted the [second primary tag][26] and the previous and/or new values for the configuration.  | `@evt.name:APM @asset.type:second_primary_tag` |
```
Comment (neko-dd) [inline content/en/account_management/audit_trail/events.md:102]:
```suggestion
| [Second primary tag][25] | A user added, modified, or deleted the [second primary tag][26] and the previous or new values for the configuration.  | `@evt.name:APM @asset.type:second_primary_tag` |
```

File: content/en/account_management/audit_trail/events.md [hunk 2]
```diff
@@ -125,11 +126,11 @@ See the [Audit Trail documentation][2] for more information on setting up and co
 ### CI Visibility events
 | Name                            | Description of audit event                                   | Query in audit explorer                                                                                               |
 |---------------------------------|--------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| [Exclusion filters][36]         | The exclusion filters have been modified.                    | `@evt.name:"CI Visibility" @asset.type:ci_app_exclusion_filters @action:modified`                                     |
+| [Quality gates rule][37]        | A user has created, modified or deleted a quality gate rule. | `@evt.name:"CI Visibility" @asset.type:ci_app_quality_gates (@action:created OR @action:modified OR @action:deleted)` |
```
Comment (neko-dd) [inline content/en/account_management/audit_trail/events.md:130]:
```suggestion
| [Quality gates rule][37]        | A user has created, modified, or deleted a quality gate rule. | `@evt.name:"CI Visibility" @asset.type:ci_app_quality_gates (@action:created OR @action:modified OR @action:deleted)` |
```

File: content/en/account_management/audit_trail/events.md [hunk 3]
```diff
@@ -140,9 +141,9 @@ See the [Audit Trail documentation][2] for more information on setting up and co
...
-| [Dashboard deleted][39] | A dashboard is deleted and the previous JSON value for the dashboard.                              | `@evt.name:Dashboard @asset.type:dashboard @action:deleted`             |
-| [Dashboard embedded][40] (Roadie) | A Datadog dashboard is [embedded into a third party][41] and a user views the dashboard.                      | `@evt.name:Dashboard @asset.type:embed @action:accessed`                |
+| [Dashboard embedded][40] (Roadie) | A Datadog dashboard is embedded into a third party ([Roadie][41]) and a user views the dashboard.                      | `@evt.name:Dashboard @asset.type:embed @action:accessed`                |
 | [Dashboard modified][42] | A dashboard is modified and the previous and new JSON values for the dashboard.                   | `@evt.name:Dashboard @asset.type:dashboard @action:modified`            |
+| [Dashboard deleted][39] | A dashboard is deleted and the previous JSON value for the dashboard.                              | `@evt.name:Dashboard @asset.type:dashboard @action:deleted`             |
```
Comment (neko-dd) [inline content/en/account_management/audit_trail/events.md:146]:
```suggestion
| [Dashboard deleted][39] | A dashboard is deleted. Also provides the previous JSON value for the dashboard.                              | `@evt.name:Dashboard @asset.type:dashboard @action:deleted`             |
```

File: content/en/account_management/audit_trail/events.md [hunk 4]
```diff
@@ -140,9 +141,9 @@ See the [Audit Trail documentation][2] for more information on setting up and co
...
 | [Dashboard created][38] | A dashboard is created and the new JSON value for the dashboard.                                    | `@evt.name:Dashboard @asset.type:dashboard @action:created`             |
-| [Dashboard deleted][39] | A dashboard is deleted and the previous JSON value for the dashboard.                              | `@evt.name:Dashboard @asset.type:dashboard @action:deleted`             |
-| [Dashboard embedded][40] (Roadie) | A Datadog dashboard is [embedded into a third party][41] and a user views the dashboard.                      | `@evt.name:Dashboard @asset.type:embed @action:accessed`                |
+| [Dashboard embedded][40] (Roadie) | A Datadog dashboard is embedded into a third party ([Roadie][41]) and a user views the dashboard.                      | `@evt.name:Dashboard @asset.type:embed @action:accessed`                |
 | [Dashboard modified][42] | A dashboard is modified and the previous and new JSON values for the dashboard.                   | `@evt.name:Dashboard @asset.type:dashboard @action:modified`            |
```
Comment (neko-dd) [inline content/en/account_management/audit_trail/events.md:145]:
```suggestion
| [Dashboard modified][42] | A dashboard is modified. Also provides the previous and new JSON values for the dashboard.                   | `@evt.name:Dashboard @asset.type:dashboard @action:modified`            |
```

File: content/en/account_management/audit_trail/events.md [hunk 5]
```diff
@@ -140,9 +141,9 @@ See the [Audit Trail documentation][2] for more information on setting up and co
...
+| [Dashboard deleted][39] | A dashboard is deleted and the previous JSON value for the dashboard.                              | `@evt.name:Dashboard @asset.type:dashboard @action:deleted`             |
 | [Dashboard user(s) added][43] | A user added user ID(s) that can access a dashboard and the list of new user IDs.                 | `@evt.name:Dashboard @asset.type:dashboard_share_acl @action:created`   |
 | [Dashboard user(s) deleted][44] | A user deleted user ID(s) that can access a dashboard and the list of the deleted user ID(s).       | `@evt.name:Dashboard @asset.type:dashboard_share_acl @action:deleted`   |
```
Comment (neko-dd) [inline content/en/account_management/audit_trail/events.md:148]: Suggestion: change any instances of `ID(s)` to `IDs` in this document. The latter tends to machine translate better IIRC and is just about as clear.

File: content/en/account_management/audit_trail/events.md [hunk 6]
```diff
@@ -160,8 +161,8 @@ See the [Audit Trail documentation][2] for more information on setting up and co
 
 | Name     | Description of audit event                                          | Query in audit explorer                           |
 | -------- | ------------------------------------------------------------------- | --------------------------------------------------|
-| [Error Tracking for Logs activation][50] | A user has enabled or disabled Error Tracking for Logs product. | `@evt.name:"Error Tracking" @action:(created OR deleted) @asset.type:error_tracking_logs` |
 | [Create or Modify inclusion filter][51] | A user has added or modified an inclusion filter. | `@evt.name:"Error Tracking" @asset.type:error_tracking_inclusion_filter` |
+| [Error Tracking for Logs activation][50] | A user has enabled or disabled Error Tracking for Logs product. | `@evt.name:"Error Tracking" @action:(created OR deleted) @asset.type:error_tracking_logs` |
```
Comment (neko-dd) [inline content/en/account_management/audit_trail/events.md:165]:
```suggestion
| [Error Tracking for Logs activation][50] | A user has enabled or disabled Error Tracking for the Logs product. | `@evt.name:"Error Tracking" @action:(created OR deleted) @asset.type:error_tracking_logs` |
```

File: content/en/account_management/audit_trail/events.md [hunk 7]
```diff
@@ -193,25 +194,25 @@ See the [Audit Trail documentation][2] for more information on setting up and co
 | Name | Description of audit event                                          | Query in audit explorer                           |
 | ---- |------------------------------------------------------------------- | --------------------------------------------------|
 | [Custom metric created][65] | A user created a custom metric and the new value for the custom metric configuration. | `@evt.name:Metrics @asset.type:metric @action:created` |
-| [Custom metric deleted][66] | A user deleted a custom metric and the previous value for the custom metric configuration. | `@evt.name:Metrics @asset.type:metric @action:deleted` |
 | [Custom metric modified][67] | A user modified a custom metric and the previous and new values for the custom metric configuration. | `@evt.name:Metrics @asset.type:metric @action:modified` |
+| [Custom metric deleted][66] | A user deleted a custom metric and the previous value for the custom metric configuration. | `@evt.name:Metrics @asset.type:metric @action:deleted` |
```
Comment (neko-dd) [inline content/en/account_management/audit_trail/events.md:198]:
```suggestion
| [Custom metric deleted][66] | A user deleted a custom metric. Also provides the previous value for the custom metric configuration. | `@evt.name:Metrics @asset.type:metric @action:deleted` |
```

File: content/en/account_management/audit_trail/events.md [hunk 8]
```diff
@@ -193,25 +194,25 @@ See the [Audit Trail documentation][2] for more information on setting up and co
...
 
 ### Monitor events
 
 | Name             | Description of audit event                                           | Query in audit explorer                                  |
 | ---------------- | -------------------------------------------------------------------- | ---------------------------------------------------------|
 | [Monitor created][68]  | A monitor is created and the new JSON value for the monitor.                 | `@evt.name:Monitor @asset.type:monitor @action:created`  |
-| [Monitor deleted][69]  | A monitor is deleted and the previous JSON value for the monitor.           | `@evt.name:Monitor @asset.type:monitor @action:deleted`  |
 | [Monitor modified][70] | A monitor is modified and the previous and new JSON values for the monitor. | `@evt.name:Monitor @asset.type:monitor @action:modified` |
+| [Monitor deleted][69]  | A monitor is deleted and the previous JSON value for the monitor.           | `@evt.name:Monitor @asset.type:monitor @action:deleted`  |
```
Comment (neko-dd) [inline content/en/account_management/audit_trail/events.md:206]:
```suggestion
| [Monitor deleted][69]  | A monitor is deleted. Also provides the previous JSON value for the monitor.           | `@evt.name:Monitor @asset.type:monitor @action:deleted`  |
```

File: content/en/account_management/audit_trail/events.md [hunk 9]
```diff
@@ -236,8 +237,8 @@ See the [Audit Trail documentation][2] for more information on setting up and co
 ### Security Notification events
 | Name                 | Description of audit event                                                       | Query in audit explorer                                           |
 | -------------------- | -------------------------------------------------------------------------------- | ------------------------------------------------------------------|
-| [Token leaked][81] | Datadog has detected leaked Datadog API or Application Key that should be revoked.| `@evt.name:"Security Notification" @asset.type:(api_key OR application_key) @action:notification` |
 | [Login method override][82] | Datadog has detected a user login method override that is different from the default login methods set for the organization.| `@evt.name:"Security Notification" @asset.type:user @action:notification` |
+| [Token leaked][81] | Datadog has detected leaked Datadog API or Application Key that should be revoked.| `@evt.name:"Security Notification" @asset.type:(api_key OR application_key) @action:notification` |
```
Comment (neko-dd) [inline content/en/account_management/audit_trail/events.md:241]:
```suggestion
| [Token leaked][81] | Datadog has detected a leaked Datadog API or Application Key that should be revoked.| `@evt.name:"Security Notification" @asset.type:(api_key OR application_key) @action:notification` |
```

**Review by neko-dd — COMMENTED** [review]
Global comment, since I realized there were many that are not highlighted as changes:

Change anything of the form `A monitor is deleted and the previous JSON value for the monitor.` to `A monitor is deleted. Also provides the previous JSON value for the monitor.` (provided that the latter is accurate).

**Comment by maycmlee** [comment]
> Change anything of the form `A monitor is deleted and the previous JSON value for the monitor.` to `A monitor is deleted. Also provides the previous JSON value for the monitor.` (provided that the latter is accurate).

Thanks @neko-dd! I'll open a separate PR to make those changes because there are 27 instances, and just want to get the reorder out first for the PM. I also want to check in on the usage of `and/or` in "..the previous and/or new values for..` sentences and see what users are seeing in the app.

**Review by neko-dd — APPROVED** [review]

**Pull request merged**